plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -401,11 +401,17 @@ class GesturesPluginTest {
     every {
       moveGestureDetector.focalPoint
     } returns PointF(0.0f, 0.0f)
-    every { moveGestureDetector.pointersCount } returns 2
+    every { moveGestureDetector.pointersCount } returns 3
     var handled = presenter.handleMove(moveGestureDetector, 50.0f, 50.0f)
+    // verify three finger pan gesture shouldn't work
     assertFalse(handled)
+    every { moveGestureDetector.pointersCount } returns 2
```

Comment:
will be nice to rewrite it as parametrized tests

Reply:
We will only have these 3 cases, and I don't foresee we will add more in the future, so would be extra overhead to rewrite it to parameterised tests imo.